### ISSUE TITLE
update waSCC demos to use wash

### ISSUE THREAD

**Issue by bacongobbler** [deislabs/krustlet#472]
Moving forward, [wash](https://github.com/wasmcloud/wash) will be the way to sign and push modules to a registry for the waSCC provider. We should update the current demos to move away from `nk` and `wascap` to the new tooling once it's stable and ready for use.

@brooksmtownsend is wash ready for use or should we continue to use `nk` and `wascap` for the time being?

**Comment by brooksmtownsend** [comment]
@bacongobbler `wash` is ready for use, the same functionality carries over from `nk` and `wascap` with some additional features that allow for autogeneration of nkeys while signing actors. The only thing that you might want to wait for is for us to get rid of all of the `git` dependencies in the project. This just depends on the soon-to-be released wasmcloud 0.15.0, and would let you recommend people running demos to install `wash` with `cargo` without specifying `--git` and `--branch`. Up to you there though, the current install command is 
```
cargo install wash-cli --git https://github.com/wasmcloud/wash --branch main
```

**Comment by thomastaylor312** [comment]
Currently handling this within the new repo
